README.md

```diff
@@ -34,3 +34,77 @@
 ### ./migration-runner.sh -e aat -o exportrecursiveexception -i realtime
 
 
+# Uploading CVS result files to Azure Storage
+
+A script is provided that allowds upload of files or directory contents to Azure storage
+
+## Pre-requisites
+
+Please make sure azure cli >= `2.0.67`
+
+This script has been tested with azure cli version `2.25.0`
+
+### Upgrade to latest version of Azure CLI
+Upgrade Azure CLI to the latest version
+
+```
+brew update
+brew upgrade azure-cli
+```
+
+### Install `storage-preview` azure cli extension
+
+`az extension add --name storage-preview`
+
+## Script Usage
+
+To upload files or directories to Azure storage use the following script 
+
+`uploadToAzureStorage.sh src dest connectionString`
+
+with following parameters
+
+- `src`
+    - the source file(s), or directory containing the files to be uploaded
+
+- `dest`
+    - the destination path on azure storage where file(s)/directory will be uploaded
+
+- connectionString
+    - used to authenticate with azure storage, should be in the following format, where ACCOUNT_NAME and ACCOUNT_KEY
+    are obviously relevant values for your azure storage container
+        
+        `DefaultEndpointsProtocol=https;AccountName=<ACCOUNT_NAME>;AccountKey=<ACCOUNT_KEY>;EndpointSuffix=core.windows.net`
+    
+### Usage examples
+
+Store the connection string in an environment variable rather than having to supply on command line each time
+
+```
+export UPLOAD_SCRIPT_CONN_STRING=`DefaultEndpointsProtocol=https;AccountName=<ACCOUNT_NAME>;AccountKey=<ACCOUNT_KEY>;EndpointSuffix=core.windows.net`
+```
+
+- Upload all csv files in local `SCSS` directory generated on 23rd June 2021 to azure storage root directory
+
+    `./uploadToAzureStorage.sh './SSCS/*-20210623-*.csv' '.' "$UPLOAD_SCRIPT_CONN_STRING"`
+
+- Upload a local single file to azure storage root directory
+    
+    `./uploadToAzureStorage.sh './CIVIL-20210621-120339.csv' './CIVIL-20210621-120339.csv' "$CONN_STR"`
+
+- Upload a local directory and all of its contents to storage root directory
+
+    `./uploadToAzureStorage.sh './SSCS/*' '.' "$CONN_STR"`
```

Comment:
"$UPLOAD_SCRIPT_CONN_STRING" ?? or you can use that env variable inside script as well rather than taking from user again.

Reply:
Resolved in https://github.com/hmcts/case-document-metadata-migration/pull/16/commits/05be2ce4c7880259b633b923fbe1b31b8d6fe5d3